bin/get_metadata.py

```diff
@@ -174,17 +188,24 @@ def get_demultiplexing_info(self):
         "--runfolder", type=str, required=True, help="Path to runfolder"
     )
     parser.add_argument(
-        "--bcl2fastq-outdir",
+        "--demultiplexer",
+        type=str,
+        default="bcl2fastq",
+        help="Name of demultiplexer used",
+    )
+    parser.add_argument(
+        "--demultiplexer-outdir",
         type=str,
-        default="Data/Intensities/BaseCalls",
-        help="Path to bcl2fastq output folder relative to the runfolder",
+        default="Unaligned",
```

Comment:
I think we want to keep `Data/Intensities/BaseCalls` as the default output directory to make this pipeline easier by other users.